tutorials/io/runtime_file_loading_and_saving.rst

```diff
@@ -168,10 +168,10 @@ Example of loading an image and displaying it in a :ref:`class_TextureRect` node
 Audio/video files
 -----------------
 
-Godot supports loading Ogg Vorbis audio at runtime. Note that not *all* files
-with an ``.ogg`` extension may be Ogg Vorbis files. Some may be Ogg Theora
-videos, or contain Opus audio within an Ogg container. These files will **not**
-load correctly as audio files in Godot.
+Godot supports loading Ogg Vorbis audio at runtime.
+
+.. note::
+    Note that not *all* files with an ``.ogg`` extension may be Ogg Vorbis files.         Some may be Ogg Theora videos, or contain Opus audio within an Ogg container.         These files will **not** load correctly as audio files in Godot.
```

Comment:
```suggestion
    Not *all* files with an ``.ogg`` extension may be Ogg Vorbis files. Some may be
    Ogg Theora videos, or contain Opus audio within an Ogg container. These files
    will **not** load correctly as audio files in Godot.
```
The formatting got a bit mangled, this should fix it.

I think if this note block is moved to after the examples and before the note about WAV and MP3, it interrupts the flow a bit less.

Reply:
About note about WAV and MP3. After merging https://github.com/godotengine/godot/pull/97855 and https://github.com/godotengine/godot/pull/97849 and similar PR for wav (which isn't ready because I don't know if @MaxIsJoe is going to implement for themselves) this notee will become outdated and wrong so I decided to convert it to draft and ship all the changes to this section in batch...

Reply:
It's okay to make smaller self-contained docs PRs. If you think this PR is worth it on its own, I would leave it up, and make a second PR for the WAV and MP3 changes when they are approved. But one big PR also works.